cfApplication.go

```diff
@@ -23,8 +23,8 @@ type Application struct {
 	// fail the deployment of the application. By default its 60 seconds.
 	// https://github.com/cloudfoundry/docs-dev-guide/blob/96f19d9d67f52ac7418c147d5ddaa79c957eec34/deploy-apps/large-app-deploy.html.md.erb#L35
 	StartupTimeout uint `json:"startupTimeout,omitempty"`
-	// Replicas configures the number of Cloud Foundry application instances.
-	Replicas uint `json:"replicas"`
+	// Instances configures the number of Cloud Foundry application replicas.
+	Instances uint `json:"instances"`
```

Comment:
@jordigilh my bad, I missed that we used `Replicas` and I'm ok with that. I can revert the [change](https://github.com/jordigilh/konveyor-enhancements/pull/2#discussion_r1924246729) in the enhancement. WDYT?

Reply:
Let's revert and I will remove the last commit

Reply:
Done. It's `Replicas` again